tools/mine-localisation-files.py

```diff
@@ -0,0 +1,208 @@
+#!/usr/bin/env python3
+#
```

Comment:
license information?

Reply:
Thanks, yes, this should have been there. I've added it in.